bat-utils/lib/extras-utils.js

```diff
@@ -39,3 +57,11 @@ function createdTimestamp (id) {
 function uint8tohex (arr) {
   return [].slice.call(arr).map((b) => ('00' + b.toString(16)).substr(-2)).join('')
 }
+
+function toBat (probi) {
```

Comment:
this belongs better in `runtime-currency.js`, but since it isn't used I would suggest removing it